tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/temporal/TableauEncoder.scala

```diff
@@ -140,6 +141,8 @@ class TableauEncoder(
                 "Expected to find no LET-IN expressions. They should have been rewritten by the inliner.")
           case OperEx(oper, args @ _*) =>
             val nodeIdentifier = TableauEncoder.NAME_PREFIX + gen.newName()
+            NameReplacementMap.store = NameReplacementMap.store
```

Comment:
Since these assignments are mutable, I wouldn't think that the assignment to `store` is necessary. Is it?